src/xtgeo/grid3d/_grid_import.py

```diff
@@ -2,11 +2,11 @@
 from __future__ import annotations
 
 from pathlib import Path
-from typing import Any, Literal
+from typing import Any
 
 from xtgeo.common import null_logger
 from xtgeo.grid3d import _grid_import_ecl, _grid_import_roff
-from xtgeo.io._file_wrapper import FileWrapper
+from xtgeo.io._file import FileFormat, FileWrapper
```

Comment:
It should work as this I would think?

`from xtgeo.io. import FileFormat, FileWrapperfrom`